web/themes/custom/nysenate_theme/src/templates/views/views-view-unformatted--microsite_events_landing.html.twig

```diff
@@ -0,0 +1,42 @@
+{#
+/**
+ * @file
+ * Theme override to display a view of unformatted rows.
+ *
+ * Available variables:
+ * - title: The title of this group of rows. May be empty.
+ * - rows: A list of the view's row items.
+ *   - attributes: The row's HTML attributes.
+ *   - content: The row's content.
+ * - view: The view object.
+ * - default_row_class: A flag indicating whether default classes should be
+ *   used on rows.
+ *
+ * @see template_preprocess_views_view_unformatted()
+ */
+#}
+<div>
+  <div class="c-container--header c-events--landing">
+    <h2 class="c-container--title">Upcoming Events</h2>
+    <a href="{{ cta_url }}" class="c-container--link">Go to Events</a>
+  </div>
+  {% for row in rows %}
+
+      {{- row.content -}}
```

Comment:
I'm happy to see the whitespace modifiers being used.  We need to use them more often.

https://twig.symfony.com/doc/1.x/templates.html#templates-whitespace-control